workflows/templates/ZWESECUR.vtl

```diff
@@ -230,13 +230,13 @@
 /*   enabled.                                                        */
 
 /* group for administrators                                          */
-/* replace AUTOGID with GID(ADMINGID.) if AUTOGID is not enabled    */
+/* replace AUTOGID with GID(&ADMINGID.) if AUTOGID is not enabled    */
```

Comment:
This line causes jenkins build to fail because the line becomes too long after a substitution. The `&` is substituted with `&amp;` for some reason which makes the line to exceed 71 characters.

Reply:
Yes, sorry, I remember that thread now.  Will fix.

Reply:
@vit-tomica Fixed, please review.